server.R

```diff
@@ -719,6 +720,266 @@ shinyServer(function(input, output, session) {
     bayesian_model(sub = TRUE, data(), treatment_list(), input$metaoutcome, input$exclusionbox, 
                    outcome_measure(), input$modelranfix, reference_alter())
   })
+  
+  # Hide the outdated Bayesian visualisations when the data is changed 
+  # or the inputs to the outcome, treatment ranking or model radio buttons are changed
+  observeEvent(c(data(), input$outcomeCont, input$outcomebina, input$rankopts, input$modelranfix), {
+    # Elements in tab 3a left column
```

Comment:
There's a lot of bloat added to the code by adding all of these IDs. Can you factor them out into a separate file which assigns lists of IDs to be hidden by all changes, and to be hidden by filters, etc.
The code is `server.R` would then just use those lists within these blocks, either iterating over them, or preferably passing the whole list into the show/hide function, if they support that.